app/screens/Menu/ManageProfiles/ManageProfiles.tsx

```diff
@@ -0,0 +1,64 @@
+import _ from 'lodash';
+import React from 'react';
+import { SafeAreaView, View } from 'react-native';
+import { useTranslation } from 'react-i18next';
+
+import { IManageProfilesScreenProps } from './ManageProfiles.types';
+import { styles } from './ManageProfiles.styles';
+import { generalStyles } from '../../../utils/styles';
+import {
+  BottomButtonView,
+  ExtendedText,
+  MainHeader,
+} from '../../../components';
+import { IMAGES } from '../../../assets';
+import { useAppSelector } from '../../../hooks';
+import { UserImageTitle } from './components';
+
+export const ManageProfilesScreen: React.FC<IManageProfilesScreenProps> = ({
+  navigation,
+}) => {
+  const { t } = useTranslation();
+
+  const parentNickname = useAppSelector(
+    state => state.user.parent?.nickname,
+  ) as string;
+  const childNickname = useAppSelector(
+    state => state.user.child?.nickname,
+  ) as string;
+
+  const childAvatar = useAppSelector(
+    state => state.user.child?.avatar as keyof typeof IMAGES,
+  );
+  const parentAvatar = useAppSelector(
+    state => state.user.parent?.avatar as keyof typeof IMAGES,
+  );
```

Comment:
Refactor it with common interface
+ make separate images for avatars @emberglaz